test/integration/full-content/full-content.spec.js

```diff
@@ -223,7 +223,7 @@ describe( 'full post content fixture', () => {
 			.map( ( block ) => block.name )
 			// We don't want tests for each oembed provider, which all have the same
 			// `save` functions and attributes.
-			.filter( ( name ) => name.indexOf( 'core-embed' ) !== 0 )
+			.filter( ( name ) => name.indexOf( 'core-embed' ) !== 0 && name !== 'core/template' )
```

Comment:
Should we update the comment here too? Not clear why we need to exclude it.

Reply:
I didn't want to add fixtures for this block. I thought this test would bring too much compared to the e2e test